CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Contributing
+Learn what you can to to help this project succeed!
+
+## How to contribute
+* **Developer** · Build the plugin from source and fix a problem from the «Known Issues» in the readme.
+* **Developer** · Build the plugin from source and go to the [issues](https://github.com/johnboiles/obs-mac-virtualcam/issues) and fix something
+* **Developer** · Build the plugin from source and do a Pull Request for a CI. For example for [Travis CI](https://travis-ci.org/). The built plugin should end up in the Github-Release of this Repo. 
+* **Tec People** · Suggest a readme/file with «Common Problems». People who try the plugin have sometimes problems with for example permission rights. See [issue #35](https://github.com/johnboiles/obs-mac-virtualcam/issues/35). Keep an eye on the issues and add sections if necessary.
+* **Tec People** · Watch this repo and whenever people come with questions in the [issues](https://github.com/johnboiles/obs-mac-virtualcam/issues) try to be the first to answer it. Even if you are not a developer you can help people by pointing in the right direction. It helps the developer to focus on writing code.
```

Comment:
I.E. Everyone should be more like you and @gxalpha :D

Thanks for all the help y'all. It really does help me focus on writing the code.